pkgs/development/libraries/openldap/default.nix

```diff
@@ -16,9 +16,14 @@ stdenv.mkDerivation rec {
   buildInputs = [ openssl cyrus_sasl db groff libtool ];
 
   configureFlags =
-    [ "--enable-overlays"
+    [
       "--disable-dependency-tracking"   # speeds up one-time build
+      "--disable-ndb"
+      "--disable-perl"
+      "--disable-sql"
+      "--enable-backends=mod"
```

Comment:
Put backends to modular load (default: yes) is problematic. If you load a module (e.g. back_hdb), you have to specify the path of modules-dir. This path is in a nix-store.
If we need this configure option, we have to add something like this
`ln -s /nix/store/hv58j5qilznnc3arjgdzymkq58cxn9wc-openldap-2.4.44/libexec/openldap /usr/lib/ldap`
to the openldap-service of NixOS. Then we can use a static path to load modules in config-files (slapd.conf) or with config-dir (slap.d).
But I think `/usr/lib/ldap` (default) isn't a good idea. Maybe `/var/db/openldap/modules` is a better choise